dandi/tests/test_download.py

```diff
@@ -140,29 +140,32 @@ def test_download_multiple_files(monkeypatch, tmpdir):
     # to this test will test those retries.
     # While at it we will also test girder downloadFile to retry at least 3 times
     # in case of some errors, and that it sleeps between retries
-    orig_downloadFileAsIterator = GirderCli.downloadFileAsIterator
+    orig_sendRestRequest = GirderCli.sendRestRequest
 
     class Mocks:
         ntries = 0
         sleeps = 0
 
         @staticmethod
-        def downloadFileAsIterator(self, *args, **kwargs):
-            Mocks.ntries += 1
-            if Mocks.ntries < 3:
-                raise gcl.HttpError(
-                    text="Failing to download", url=url, method="GET", status=500
-                )
-            return orig_downloadFileAsIterator(self, *args, **kwargs)
+        def sendRestRequest(self, *args, **kwargs):
+            if (
+                len(args) > 1
+                and args[1].startswith("file/")
+                and args[1].endswith("/download")
+            ):
+                Mocks.ntries += 1
+                if Mocks.ntries < 3:
+                    raise gcl.HttpError(
+                        text="Failing to download", url=url, method="GET", status=500
+                    )
+            return orig_sendRestRequest(self, *args, **kwargs)
```

Comment:
if any of our test files big enough so we could trigger redownload when somewhere past `2*chunk` sizes?

Reply:
No.  Both of the files in `dandi/tests/data/dandifiles` are under a megabyte.

Reply:
Let's make a fixture producing sizeable file for local upload/download testing as I have outlined in the review summary?